### DISCUSSION TITLE
How to hide schema section of openapi

### DISCUSSION THREAD

**Discussion by jesumyip** [fastapi/fastapi#3861] [Questions] [answered]
Hi

I have multiple schema sections published which I wish to hide. Or maybe I am doing it the wrong way. I have more than one api that uses the same enum. I notice in the Swagger docs, at the schema section, the same enum appears multiple times - once for each time I use it in an API. So if I have 5 apis using it, it appears 5 times. 

1. Is there a way I can make it appear only once?
2. Or how can I completely hide the entire schema section at the bottom?

**Comment by jesumyip** [comment, marked as answer]
Nevermind. I figured out my mistake.

**Comment by ghandic** [comment]
Is this closed @jesumyip ?

**Reply by jesumyip** [reply]
Yes
